doc/source/structures/vessels/gimbal.rst

```diff
@@ -35,11 +43,27 @@ Many engines in KSP have thrust vectoring gimbals and in ksp they are their own
         * - :attr:`ROLLANGLE`
           - scalar
           - Current Gimbal Roll 
-		  
-        * - :attr:`LOCK`
-          - boolean
-          - Is the gimbal free to travel? 
-		  
+
+
+.. note::
+
+    :struct:`Gimbal` is a type of :struct:`PartModule`, and therefore can use all the suffixes of :struct:`PartModule`. Shown below are only the suffixes that are unique to :struct:`Gimbal`.
```

Comment:
Done! Also fixed accidentally misplaced table heading when moving the editable fields upward